src/olympia/addons/tests/test_serializers.py

```diff
@@ -1049,9 +1051,7 @@ def serialize(self):
 
         obj = self.search()
 
-        with self.assertNumQueries(0):
-            result = self.serializer.to_representation(obj)
-        return result
+        return self.serializer.to_representation(obj)
```

Comment:
I know this is just a draft & not finished yet, but FYI, the search & autocomplete APIs handle a lot of our traffic and should *never* execute any database queries. We shouldn't ship a change that would regress this `assertNumQueries(0)` assertion and the one below, they are important to keep.